04week/mastermind.js

```diff
@@ -28,15 +28,33 @@ function getRandomInt(min, max) {
   return Math.floor(Math.random() * (max - min)) + min;
 }
 
-function generateHint() {
+function generateHint(guess) {
   // your code here
+  for(let i = 0; i < solution.length;i++) {
```

Comment:
this will not correctly handle a solution that has repeated values. 
solution: aabb, and guess: axxx